### PR TITLE
add modify rocksdb config api

### PR DESCRIPTION
**Pull request by UncP**

### CODE REVIEW

File: proto/debugpb.proto [hunk 1]
```diff
@@ -172,3 +175,13 @@ message RegionConsistencyCheckRequest {
 
 message RegionConsistencyCheckResponse {
 }
+
+message ModifyRocksDBConfigRequest {
```
Comment (zhangjinpeng1987) [inline proto/debugpb.proto]: Please make this request can be used by other TiKV configurations.

**Comment by UncP** [comment]
@zhangjinpeng1987

**Comment by zhangjinpeng1987** [comment]
It's better to wait https://github.com/pingcap/tikv/pull/2838 to be done, or TiKV will have compile error.

**Comment by UncP** [comment]
No, this pr will fix that, we should merge this first.

**Comment by zhangjinpeng1987** [comment]
You haven't finish the job https://github.com/pingcap/tikv/pull/2838, if merge this pr now, there will be a gap that others will encounter compile error when update kvproto and compile TiKV.

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**